### PR TITLE
Updated template to include necessary resources

### PR DESCRIPTION
**Pull request by katiewest820**
https://newrelic.atlassian.net/browse/LAMBDA-1202

#### Included resources: 
- Function App
- App Service plan
- Event Hubs Namespace
- Storage account
- Application Insights
- Diagnostic Settings
- Log Analytics workspace

### CODE REVIEW

**Review by mrickard — APPROVED** [review]

File: templates/azure-log-ingestion.json [hunk 1]
```diff
@@ -8,27 +8,60 @@
       },
       "minLength": 1,
       "type": "String"
+    },
+    "appInsightsName": {
```
Comment (kolanos) [inline templates/azure-log-ingestion.json:12]: Do we want to make this configurable? Or just generate a name as a variable?

File: templates/azure-log-ingestion.json [hunk 2]
```diff
@@ -8,27 +8,60 @@
...
+      "type": "string",
+      "metadata": {
+        "description": "Name of Application Insights resource."
+      }
+    },
+    "workspacesName": {
```
Comment (kolanos) [inline templates/azure-log-ingestion.json:18]: Same question about whether we want to make it configurable.

File: templates/azure-log-ingestion.json [hunk 3]
```diff
@@ -8,27 +8,60 @@
...
+      "type": "string",
+      "metadata": {
+        "description": "Name of Workspaces resource."
+      }
+    },
+    "requestSource": {
```
Comment (kolanos) [inline templates/azure-log-ingestion.json:24]: Assume this is for debugging purposes only?

File: templates/azure-log-ingestion.json [hunk 4]
```diff
@@ -8,27 +8,60 @@
...
+      "type": "string",
+      "metadata": {
+        "description": "Source of Azure Resource Manager deployment"
+      }
+    },
+    "diagnosticSettingsName": {
```
Comment (kolanos) [inline templates/azure-log-ingestion.json:30]: Think we'll likely want to generate a name for this value.

**Review by kolanos — COMMENTED** [review]
Generally LGTM. Wondering if we should be more opinionated about names for things I don't think user's will care much to configure?

**Comment by katiewest820** [comment]
> Generally LGTM. Wondering if we should be more opinionated about names for things I don't think users will care much to configure?

Totally had the same thought! I think hardcoding a name makes sense.

**Review by MattWhelan — APPROVED** [review]

**Review by mrickard — APPROVED** [review]

**Pull request merged**